packages/css/src/iconButton/iconButton.css

```diff
@@ -0,0 +1,91 @@
+.md-icon-button {
+  align-items: center;
+  background-color: var(--mdPrimaryColor);
+  color: #ffffff;
+  font-family: 'Open sans';
+  border-radius: 50%;
+  width: 32px;
+  height: 32px;
+  justify-content: center;
+  display: flex;
+  border: none;
+}
+
+.md-icon-button__icon {
+  height: 20px;
+  width: 20px;
+  flex-shrink: 0;
+}
+
+.md-icon-button:focus {
+  background-color: var(--mdPrimaryColor80);
+  outline: 2px white solid;
+  outline-offset: -3px;
```

Comment:
Keep in mind the boxing model: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

If you switch the top level element to border-box, then you don’t need to subtract border sizes/outlines as they are not part of the size calculations. I think you should try it here as this is something New.